docs/AzureBlobStorageConnectorDoc.md

```diff
@@ -1,61 +1,179 @@
-</tbody></table>
-
-## Actions supported by the connector
-The following automated operations can be included in playbooks and you can also use the annotations to access operations from FortiSOAR&trade; release 4.10.0 and onwards:
-<table border=1><thead><tr><th>Function<br></th><th>Description<br></th><th>Annotation and Category<br></th></tr></thead><tbody><tr><td>Put Blob<br></td><td>Creates a new blob or replaces an existing blob within a container.<br></td><td>create_blob <br/>Investigation<br></td></tr>
-<tr><td>List Blob<br></td><td>The List Blob operation returns a list of the blobs under the specified container.<br></td><td>list_blob <br/>Investigation<br></td></tr>
-</tbody></table>
-
-### operation: Put Blob
-#### Input parameters
-<table border=1><thead><tr><th>Parameter<br></th><th>Description<br></th></tr></thead><tbody><tr><td>Blob Name<br></td><td>Specify the name of the blob to create or replace.<br>
-</td></tr><tr><td>File Attachment/IRI Reference<br></td><td>File Attachment/IRI Reference that is used to access the attachment metadata from the FortiSOAR™ Attachments module. In the playbook, if you select 'Attachment ID', this defaults to the {{vars.attachment_id}} value or if you select 'File IRI', then this defaults to the {{vars.file_iri}} value.<br>
-</td></tr><tr><td>Type<br></td><td>Choose between Attachment ID or a File IRI.<br>
-</td></tr></tbody></table>
-
-#### Output
-
- {
-     "status": "",
-     "message": ""
- }
-### operation: List Blob
-#### Input parameters
-None.
-#### Output
-
-{
+<h2>About the connector</h2>
+
+<p>Azure Blob Storage is Microsoft's object storage solution for the cloud. Blob Storage is optimized for storing massive amounts of unstructured data. Azure Blob Storage stores text and binary data as objects in the cloud. This connector helps you to perform REST operations for working with blobs in the Blob service.</p>
+
+<p>This document provides information about the Azure Blob Storage Connector, which facilitates automated interactions, with a Azure Blob Storage server using FortiSOAR&trade; playbooks. Add the Azure Blob Storage Connector as a step in FortiSOAR&trade; playbooks and perform automated operations with Azure Blob Storage.</p>
+
+<h3>Version information</h3>
+
+<p>Connector Version: 1.1.0</p>
+
+<p>FortiSOAR&trade; Version Tested on: 7.5.0-4015</p>
+
+<p>Azure Blob Storage Version Tested on: </p>
+
+<p>Authored By: Fortinet</p>
+
+<p>Certified: Yes</p>
+
+<h2>Release Notes for version 1.1.0</h2>
+
+<p>Following enhancements have been made to the Azure Blob Storage Connector in version 1.1.0:</p>
+
+<ul>
+    <li>Added the following new operations and playbooks:
+        <ul>
+            <li>Get Blob</li>
+            <li>Copy Blob</li>
+            <li>Delete Blob</li>
+            <li>Abort Copy Blob</li>
+            <li>Get Blob Properties</li>
+            <li>Get Blob Metadata</li>
+            <li>Get Blob Tags</li>
+        </ul>
+    </li>
```

Comment:
**Please add the following point:**
Renamed the action "Put Blob" to "Create Blob"

@cs-shivaji-kolape  Please add the same in the release-notes as well.